cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/StatefulSetRollingUpdateTest.java

```diff
@@ -39,8 +39,8 @@ public class StatefulSetRollingUpdateTest {
     @BeforeEach
     public void before() {
         KafkaVersion.Lookup versions = new KafkaVersion.Lookup(emptyMap(), emptyMap(), emptyMap(), emptyMap());
-        currectSts = KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, getResource(), versions).generateStatefulSet(true, null, null, null);
-        desiredSts = KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, getResource(), versions).generateStatefulSet(true, null, null, null);
+        currectSts = KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, getResource(), versions).generateStatefulSet(3, true, null, null, null);
+        desiredSts = KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, getResource(), versions).generateStatefulSet(3, true, null, null, null);
```

Comment:
Here and in other similar places, you should probably move the replica count to some constant instead of having it defined five times. Or you can make it more versatile and take it from the Kafka resource